admin/js/boldgrid-backup-admin-settings-autoupdate.js

```diff
@@ -66,51 +89,68 @@ BOLDGRID.SETTINGS = BOLDGRID.SETTINGS || {};
 				$pluginToggles = $bgBox.find( '.plugin-toggle' ),
 				$themeToggles = $bgBox.find( '.theme-toggle' ),
 				$pluginsDefault = $bgBox.find( '#toggle-default-plugins' ),
-				$themesDefault = $bgBox.find( '#toggle-default-themes' );
+				$themesDefault = $bgBox.find( '#toggle-default-themes' ),
+				$timelyUpdatesEnabled = $bgBox.find( '#timely-updates-enabled' ),
+				$timelyUpdatesDisabled = $bgBox.find( '#timely-updates-disabled' ),
+				$timelyUpdatesDays = $bgBox.find( '#timely-updates-days' );
 
 			// If the updates section is not in use, then just return.
-			if ( ! $pluginsDefault.data( 'toggles' ) ) {
-				return;
-			}
+			// if ( ! $pluginsDefault.data( 'toggles' ) ) {
+			// 	return;
+			// }
 
 			$wpcoreToggles.each( function() {
 				var $this = $( this );
 
-				$this
+				var $thisInput = $this
 					.next( 'input' )
-					.attr( 'name', 'autoupdate[wpcore][' + $this.data( 'wpcore' ) + ']' )
+					.attr( 'name', 'auto_update[wpcore][' + $this.data( 'wpcore' ) + ']' )
 					.val( $this.data( 'toggles' ).active ? 1 : 0 );
+				$this.change( function() {
+					if ( true == $this.prop( 'checked' ) ) {
+						$thisInput.val( '1' );
+					} else {
+						$thisInput.val( '0' );
+					}
+				} );
 			} );
 
 			$pluginToggles.each( function() {
 				var $this = $( this );
 
-				$this
-					.parent()
+				var $thisInput = $this
 					.next( 'input' )
-					.attr( 'name', 'autoupdate[plugins][' + $this.data( 'plugin' ) + ']' )
+					.attr( 'name', 'auto_update[plugins][' + $this.data( 'plugin' ) + ']' )
 					.val( $this.data( 'toggles' ).active ? 1 : 0 );
+
+				$this.change( function() {
+					if ( true == $this.prop( 'checked' ) ) {
+						$thisInput.val( '1' );
+					} else {
+						$thisInput.val( '0' );
+					}
+				} );
 			} );
 
 			$themeToggles.each( function() {
 				var $this = $( this );
 
-				$this
-					.parent()
+				var $thisInput = $this
 					.next( 'input' )
-					.attr( 'name', 'autoupdate[themes][' + $this.data( 'stylesheet' ) + ']' )
+					.attr( 'name', 'auto_update[themes][' + $this.data( 'stylesheet' ) + ']' )
```

Comment:
Why was it changed from `autoupdate` to `auto_update`?

Reply:
i liked it better with the underscore when storing it in the boldgrid backup settings. For everything to play nicely together, it was easiest if the input's name matches the setting.